students/alxsk/session10/virtualenv/musical_comprehensions.py

```diff
@@ -0,0 +1,11 @@
+'''musical comprehensions
+ Find top 5 artists and song name for track (sort desc) with danceability >.8 and loudness <5.0'''
+
+import pandas as pd
+music = pd.read_csv('music.csv')
+
+
+print(music[music.loudness > -5].sort_values(by = 'danceability', ascending = False).head(5))
```

Comment:
I think you might want to expand this out to sorting on loudness and danceability. Also, how can you do this using a list comprehension and the sorted function?